packages/api-v2/src/hooks/useDerivAccountsList.ts

```diff
@@ -1,18 +1,25 @@
 import { useMemo } from 'react';
+import useQuery from '../useQuery';
 import useAuthorize from './useAuthorize';
 import useBalance from './useBalance';
 import useCurrencyConfig from './useCurrencyConfig';
 import { displayMoney } from '../utils';
 
 /** A custom hook that returns the list of accounts for the current user. */
 const useDerivAccountsList = () => {
-    const { data: authorize_data, ...rest } = useAuthorize();
+    const { data: authorize_data, isSuccess } = useAuthorize();
+    const { data: account_list_data, ...rest } = useQuery('account_list', {
+        options: {
+            enabled: isSuccess,
+            refetchOnWindowFocus: false,
```

Comment:
why do we set refetchOnWindowFocus to false

Reply:
To minimize the number of `account_list` requests made.